### ISSUE TITLE
Reconciliation button is missing for columns annotated as xsd:string

### ISSUE THREAD

**Issue by vcutrona** [datagraft/grafterizer-2.0#91]
Reconciliation is prevented on xsd:string columns.

How to reproduce:

1. Create a new transformation and upload a file
2. Switch to the Tabular Enrichment tab
3. Annotate a column (**A**) as a literal column of type string.

**RESULT**: The Reconciliation button disappeared from **A**.

**Comment by nvnikolov** [comment]
This is by design - literals cannot be reconciled (literals in RDF cannot have an rdf:type).

**Comment by vcutrona** [comment]
This is not properly right. Reconciliation always happens on strings, and the new (reconciled) column will be annotated with the rdf:type of the reconciled entities.
We should rather prevent reconciliation on columns annotated with other types (e.g., date or number).

**Comment by nvnikolov** [comment]
You are right about strings, but not about _RDF literal strings_. If you mark something as a literal (of any type) you explicitly define that it can not be a base of a triple. Also, all columns are treated as strings by default

**Comment by vcutrona** [comment]
In tabular annotation, it works the other way round. If you use an RDF literal string column as the starting point of reconciliation, that column will be the object of a triple, while the subject is represented by the new reconciled column. The question is how to backlink the object column (xsd:string) to its source (reconciled) column (the rdfs:label property might be a good default).

**Comment by nvnikolov** [comment]
Thanks - I understand now. 

The backlink should ideally happen with the property that the reconciliation service has identified as linking the reconciled entity with the matched literal (I guess you mostly look at "rdfs:label" properties, but it could be something else depending on the service)
